bot/plotting/comparison_generator.py

```diff
@@ -93,7 +92,14 @@ def comparison_generator(
                         continue
                     else:
                         break
-                else:    # pragma: no cover
+                elif (
+                    param_to_vary == "Ambient temperature [K]"
+                ):    # pragma: no cover
+                    if param_value < 265 or param_value > 355:
```

Comment:
I wonder if it is cleaner to put some acceptable bounds into `parameter_value_generator` instead of doing the continue/break while loop here?

Reply:
Yes, it would be, right now the conditions look messy. I'll look into this, thanks!